crates/model_import/src/gltf/mod.rs

```diff
@@ -52,22 +51,9 @@ pub async fn import(import: &GltfImport, asset_crate: &mut ModelCrate) -> anyhow
                     .read_joints(0)
                     .map(|v| v.into_u16().map(|v| uvec4(v[0] as u32, v[1] as u32, v[2] as u32, v[3] as u32)).collect::<Vec<UVec4>>()),
                 joint_weights: reader.read_weights(0).map(|v| v.into_f32().map(|x| x.into()).collect::<Vec<Vec4>>()),
-                indices: reader.read_indices().map(|v| {
-                    if flip_indices {
-                        v.into_u32()
-                            .chunks(3)
-                            .into_iter()
-                            .flat_map(|chunk| {
-                                let mut chunk = chunk.collect_vec();
-                                chunk.swap(1, 2);
-                                chunk
-                            })
-                            .collect::<Vec<u32>>()
-                    } else {
-                        v.into_u32().collect::<Vec<u32>>()
-                    }
-                }),
+                indices: reader.read_indices().context("GLTF mesh must contain an index buffer")?.into_u32().collect_vec(),
```

Comment:
Wont this unconditionally flip the indices/winding?

Reply:
Yep, this was the case before as well, as `flip_indices` was a constant `true`.

Reply:
Oh I see now. Welp, if we need that behavior we have the history

Reply:
Yeah. If we wanted to do it properly, here's what the GLTF 2.0 spec says:

> When a mesh primitive uses any triangle-based topology (i.e., triangles, triangle strip, or triangle fan), the determinant of the node’s global transform defines the winding order of that primitive. If the determinant is a positive value, the winding order triangle faces is counterclockwise; in the opposite case, the winding order is clockwise.